spec/models/flight_spec.rb

```diff
@@ -44,4 +57,52 @@
       expect(flight).to belong_to(:destination_airport).class_name(:Airport)
     end
   end
+
+  describe "Connected Models" do
+    it { expect(flight.origin_airport).to be_a(Airport) }
+    it { expect(flight.destination_airport).to be_a(Airport) }
+  end
+
+  describe ".get_match" do
+    it "returns the flight that matches the search" do
+      match = Flight.get_match(
+        origin_airport_id: flight.origin_airport.id,
+        destination_airport_id: flight.destination_airport.id,
+        available_seats: "1",
+        departure_datetime: ""
+      ).first
+      expect(match).to eq(flight)
+    end
+  end
+
+  describe "Model scopes" do
+    it do
+      expect(Flight.not_departed.first).to eq(
+        Flight.where("departure_datetime > ?", Time.now).first
```

Comment:
Do not use `Time.now` without zone. Use one of `Time.zone.now`, `Time.now.current`, `Time.now.in_time_zone`, `Time.now.utc`, `Time.now.getlocal`, `Time.now.iso8601`, `Time.now.jisx0301`, `Time.now.rfc3339`, `Time.now.to_i`, `Time.now.to_f` instead.